include/wx/osx/app.h

```diff
@@ -139,6 +139,9 @@ class WXDLLIMPEXP_CORE wxApp: public wxAppBase
     // override this to return false from a non-bundled console app in order to stay in background ...
     virtual bool         OSXIsGUIApplication() { return true; }
 
+    // Allow the user to disable the tab bar support in the application
+    virtual void         OSXEnableAutomaticTabbing(bool enable);
```

Comment:
Does this really need to be virtual?

Reply:
I decided to make it virtual so that developers could override it and do other processing to prepare for disabling tabbing (such as setting internal states, etc.). I don't really see a downside to having it virtual.

Reply:
The downside is exactly due to the fact that we need to support overriding it, so any changes to the code later would need to deal with the fact that this function could be overridden.

Personally I don't see any point in making it virtual: if you need to do something else, why not just do it in the same code which calls this function? It's in your application (i.e. not in the library), so it's completely under your control anyhow.

Reply:
I guess since it is only doing one call into the OS, there isn't much need for overriding it after all.